<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1473</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,4">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="formAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="methodSelectorLabel">
          <property name="text">
           <string>Choose minimization method:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="methodSelector"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="epsSelectorLabel">
          <property name="text">
           <string>Desired accuracy:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QDoubleSpinBox" name="epsSelector">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Arial','Helvetica Neue','Helvetica','sans-serif'; font-size:15px; color:#242729; background-color:#fefefb;&quot;&gt;â‹…&lt;/span&gt;&lt;span style=&quot; font-family:'arial','sans-serif'; font-size:16px; color:#202124; background-color:#ffffff;&quot;&gt;10^&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="powSelector">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>-4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="arrowheadsCheckBox">
          <property name="text">
           <string>Show arrowheads</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="function">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="levelSetsCheckBox">
          <property name="text">
           <string>Show level sets</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Starting point:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>x:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="startXSelector">
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="startYSelector">
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="snap">
        <property name="text">
         <string>save screenshot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Cascadia Mono</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0062ff;&quot;&gt;ðŸ¤¡ SOFTWARE&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0062ff;&quot;&gt;DP5 KP2 NH5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>1</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="NavigableChartView" name="visualChartView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1473</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigableChartView</class>
   <extends>QGraphicsView</extends>
   <header>NavigableChartView.h</header>
   <slots>
    <slot>iterationDisplayNChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>methodSelector</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>methodChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>epsSelector</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>epsChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>powSelector</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>powChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>snap</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>screenshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>arrowheadsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleArrowheads(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelSetsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleLevelSets(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startXSelector</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>startXChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startYSelector</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>startYChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>iterationNChanged(int)</slot>
  <slot>methodChanged(int)</slot>
  <slot>epsChanged(double)</slot>
  <slot>powChanged(int)</slot>
  <slot>powEpsEditingFinished()</slot>
  <slot>multiMethodChanged(int)</slot>
  <slot>screenshot()</slot>
  <slot>toggleArrowheads(bool)</slot>
  <slot>toggleLevelSets(bool)</slot>
  <slot>startXChanged(double)</slot>
  <slot>startYChanged(double)</slot>
 </slots>
</ui>
